src/main/java/com/puppycrawl/tools/checkstyle/checks/design/MutableExceptionCheck.java

```diff
@@ -66,6 +66,136 @@
  * <pre>
  * &lt;module name=&quot;MutableException&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ *      class FirstClass extends Exception {
+ *          private int errorCode; // OK Does not extend any class
+ *
+ *          public FirstClass() {
+ *              errorCode = 1;
+ *          }
+ *      }
+ *
+ *      class MyException extends Exception {
+ *          private int errorCode; // violation , The field 'errorCode' must be declared final
```

Comment:
Why this is violation and first is ok ?
Please share CLI output for all 3 examples to prove that behavior is matching.
Share output in text form.

Reply:
it's cause in the first-one it doesn't have Exception keyword in the classname

Reply:
Ok, very unclear, let put training comment line of class name not on field. To show that we ignore such classes at all

Reply:
Done

Reply:
Done